recipe/meta.yaml

```diff
@@ -11,22 +11,26 @@ source:
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  #[py<38 or s390x]
+  # skipping s390x as it is not available for numba or pyarrow
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - python >=3.8
+    - python
     - param
+    - pyct >=0.4.4
```

Comment:
pyct doesn't seem required.

Reply:
I noticed that it was required for the build system as "pyct >=0.4.4", [here](https://github.com/holoviz/spatialpandas/blob/aeeba42751fd7cc16fd7e4a142b053fb9cd0c033/pyproject.toml#L4).

Reply:
Yep, but in practice in our case it's not used.

Reply:
Ahh got it.

Reply:
Is there a reason why?

Reply:
If you search for `pyct` references in the code (https://github.com/search?q=repo%3Aholoviz%2Fspatialpandas%20pyct&type=code), we can see that it's not directly used by any of the stuff used to create (build) the package (setup.py).

Though it doesn't harm to include it. So you can leave it there, or remove it. If you prefer to keep the dependencies exactly the same as in upstream, that's fine, but I'd add a comment to at least say why it's there.

Reply:
Oh, that's interesting. I wonder why upstream decided to do it in that way. I went ahead and removed it. Thanks for the clarification!